### PR TITLE
Iss327: Progress Report has some problems

### PR DESCRIPTION
**Pull request by JRustyHaner**
#327

### CODE REVIEW

File: mofacts/client/views/experiment/unitEngine.js [hunk 1]
```diff
@@ -229,10 +227,10 @@ function defaultUnitEngine(curExperimentData) {
       console.log('setUpCardQuestionSyllables:', currentQuestion, currentQuestionPart2,
           currentAnswerSyllables, clozeQuestionParts, currentAnswer);
       return {currentQuestionPostSylls: currentQuestion, currentQuestionPart2PostSylls: currentQuestionPart2,
-        currentAnswerSyllables, clozeQuestionParts, currentAnswer, cHintLevel};
+        currentAnswerSyllables, clozeQuestionParts, currentAnswer, hintLevel};
     },
 
-    setUpCardQuestionAndAnswerGlobals: async function(cardIndex, whichStim, probFunctionParameters) {
+    setUpCardQuestionAndAnswerGlobals: async function(cardIndex, whichStim, whichHintLevel = 0, probFunctionParameters,) {
```
Comment (andrewtackett) [inline mofacts/client/views/experiment/unitEngine.js:233]: extra comma

File: mofacts/client/views/experimentReporting/studentReporting.js [hunk 2]
```diff
@@ -123,57 +122,32 @@ Template.studentReporting.rendered = async function() {
...
+    const studentData = await meteorCallAsync('getStudentReportingData', studentID, selectedTdfId, "0");
     console.log("studentData loaded...",studentData);
-    const curStudentGraphData = await meteorCallAsync('getStudentPerformanceByIdAndTDFId',studentID,selectedTdfId,hintlevel);
+    const curStudentGraphData = await meteorCallAsync('getStudentPerformanceByIdAndTDFId',studentID,selectedTdfId,"0");
 
     console.log('studentData', studentData);
     console.log('curStudentGraphData',curStudentGraphData);
     
-    setUpHintLevelSelection(studentID, selectedTdfId);
+
```
Comment (andrewtackett) [inline mofacts/client/views/experimentReporting/studentReporting.js:143]: extra whitespace

File: mofacts/client/views/experiment/unitEngine.js [hunk 3]
```diff
@@ -196,15 +196,15 @@ function defaultUnitEngine(curExperimentData) {
...
+          const {clozeQuestion, clozeMissingSyllables, clozeQuestionParts: cQuestionParts} =
               this.replaceClozeWithSyllables(currentQuestion, currentAnswerSyllables, currentStimAnswer,hintLevel);
           if (clozeQuestion) {
             currentQuestion = clozeQuestion;
             currentAnswer = clozeMissingSyllables;
             clozeQuestionParts = cQuestionParts;
             console.log('clozeQuestionParts:', cQuestionParts);
             const {clozeQuestion2, clozeMissingSyllables2, hintlevel2} =
-                this.replaceClozeWithSyllables( currentQuestionPart2, currentAnswerSyllables, currentStimAnswer);
+                this.replaceClozeWithSyllables( currentQuestionPart2, currentAnswerSyllables, currentStimAnswer,0);
```
Comment (andrewtackett) [inline mofacts/client/views/experiment/unitEngine.js:207]: Why is this hardcoded to 0?

Comment (JRustyHaner) [reply mofacts/client/views/experiment/unitEngine.js]: Because I forgot to get some caffeine this morning. Corrected.

**Pull request merged**